demo-ui/app/uk/gov/ons/addressIndex/demoui/views/singleMatch.scala.html

```diff
@@ -27,6 +29,44 @@ <h3 class="panel-title">@messages("single.sfatext")</h3>
                 </div>
                 }
             </div>
+            @addressBySearchResponse.map { aresponse =>
+            <div class="panel panel-single address-panel">
+                <h3 class="green">@aresponse.total addresses found</h3>
+            </div>
+            @aresponse.addresses.map{ address =>
+            <div class="panel panel-match address-panel">
+                <div class="result-content">
+                    <div class="alert alert-success" role="alert">
+                        <table class="matchResults">
+                            <tr>
+                                <td class="match-found-txt"><strong>Residential</strong></td>
+                                <td style="text-align:right" class="match-score">Matching Score: @address.underlyingScore</td>
+                            </tr>
+                            <tr>
+                                <td colspan="2" class="formatted-address-txt"><strong>Address: </strong>
+                                    @address.paf.map{ paf =>
+                                        @paf.buildingName &nbsp;
+                                        @paf.buildingNumber,
+                                        @paf.thoroughfare,
+                                        @paf.dependentLocality
+                                        @paf.postTown,
+                                        @paf.postcode
+                                    }
+                                </td>
+                            </tr>
+                            <tr>
+                                <td class="uprn-txt"><strong>UPRN: </strong>&nbsp;@address.uprn</td>
+                                <td style="text-align:right" class="map-link-txt">
+                                    <a href="https://www.google.co.uk/maps/place/@address.paf.map{paf=>@paf.buildingNumber+@paf.thoroughfare+@paf.postTown+@paf.postcode.replace(' ','+')}" target="_blank">Open Map</a></td>
```

Comment:
Should look to map at higher level. If this option is `None` then it will produce an empty/dead link